code/modules/research/rdmachines.dm

```diff
@@ -39,8 +39,6 @@
 
 /obj/machinery/rnd/attackby(obj/item/O, mob/user, params)
 	if (default_deconstruction_screwdriver(user, "[initial(icon_state)]_t", initial(icon_state), O))
-		if(linked_console)
-			disconnect_console()
 		return
```

Comment:
since you added crowbar_act and such, you should be able to remove those parts of attackby